compile/core.rb

```diff
@@ -151,7 +151,10 @@ def ansible_style_yaml(obj, options = {})
     # Refer to Compile::Core.compile for full details about the compilation
     # process.
     def compile_file(ctx, source)
-      compile_string(ctx, File.read(source))
+      if !$pwd
+        $pwd = Dir.pwd
+      end
+      compile_string(ctx, File.read($pwd+"/"+source))
```

Comment:
Does this really change things?  That's so wild.

Reply:
Want to document our conversation from yesterday, but this changed because we're now cd-ing into the TPG directory and calling from there, so `$pwd` is the `magic-module` directory and this now calls the absolute path.